lib/index.ts

```diff
@@ -179,7 +179,10 @@ export async function inspect(
         cwd: mvnWorkingDirectory,
       },
     );
-    const parseResult = parse(result, options.dev);
+
+    const includesVerboseFlag = mvnArgs.includes('-Dverbose');
```

Comment:
not very likely, but what if the command is `-Dverbose=false`, which is a user trying to disable verbose

Reply:
this is a good one, we will make sure of double check that -Dverbose will be false 👍